src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -118,6 +124,7 @@ function initStore() {
                         }
 
                         const changes = updateResultsViewQuery(resultsViewPageStore.rvQuery, query, samplesSpecification, cancerStudyIds, oql);
+
```

Comment:
@adamabeshouse this initDriverAnnotations thing is yucky to me.  Idea: should we just reinstantiate the results store every time the query changes?  That way we don't have to any of this equality checking or re-init.  The client level caching will prevent refetching of data.   We just start fresh store every change to query.  It would simplify.

Reply:
yeah I completely agree

Reply:
on second thought, what if we do sometimes want to keep some state in the store but not in the url. it would be lost.  hmmmm.

Reply:
but when would the query change but the state not be reset?